Dockerfile

```diff
@@ -1,11 +1,12 @@
-FROM php:7.2-apache
-ENV TERM=xterm
+ARG PHP_VERSION=7.2
+FROM php:${PHP_VERSION}-apache
+ARG VERSION=latest
 LABEL maintainer="Derek P Sifford <dereksifford@gmail.com>" \
-      version="0.17.0-php7.2"
+      version="${VERSION}-php${PHP_VERSION}"
 
 # Install base requirements & sensible defaults + required PHP extensions
-RUN echo "deb http://ftp.debian.org/debian stretch-backports main" >> /etc/apt/sources.list \
-    && apt-get update \
+# RUN echo "deb http://ftp.debian.org/debian $(sed -n 's/^VERSION=.*(\(.*\)).*/\1/p' /etc/os-release)-backports main" >> /etc/apt/sources.list \
```

Comment:
Do you want to keep this? I don't think commented out code should stay.

Reply:
Yep, agreed. That's what the first checkbox in the PR was addressing. 

Will fix.